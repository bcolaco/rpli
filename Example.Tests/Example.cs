namespace Example.Tests;

using System.IO;
using RPLi;
using Snapshooter.Xunit;

public class Example
{
    [Fact]
    public void SimpleHTML()
    {
        var projectDirectory = Directory.GetParent(Environment.CurrentDirectory)!.Parent!.Parent!.FullName;
        var rpl = File.ReadAllText(Path.Join(projectDirectory, "example.rpl"));

        var document = RPLi.Render(rpl);

        Snapshot.Match(document);
    }
}
