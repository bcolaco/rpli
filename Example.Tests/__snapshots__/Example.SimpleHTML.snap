<html>
<body>
    <p>Hello RPL!</p>
</body>
</html>
